changelog.txt

```diff
@@ -1,3 +1,27 @@
+- Changelog for v2.2.0-rc2 (2020-11-23):
```

Comment:
No - this bit goes in RELEASE_NOTES.md - changelog is made with `make changelog`

Reply:
Honestly you could just leave the release notes off until the final - wouldn't the the first time.